clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleHttpLoadBalancerCachingAgent.groovy

```diff
@@ -335,7 +335,7 @@ class GoogleHttpLoadBalancerCachingAgent extends AbstractGoogleLoadBalancerCachi
 
       // Process queued backend services.
       queuedServices?.each { backendServiceName ->
-        BackendService service = projectBackendServices.find { bs -> Utils.getLocalName(bs.getName()) == backendServiceName }
+        BackendService service = projectBackendServices?.find { bs -> Utils.getLocalName(bs.getName()) == backendServiceName }
```

Comment:
Does `getLocalName` crash if passed `null`?

Reply:
No, it returns null.